### PR TITLE
Add Kenyan calendar

### PR DESCRIPTION
**Pull request by KidkArolis**
- [x] Tests with a significant number of years to be tested for your calendar.
- [x] Docstrings for the Calendar class and specific methods.
- [x] Use the ``workalendar.registry_tools.iso_register`` decorator to register your new calendar using ISO codes (optional).
- [x] Calendar country / label added to the README.rst file.
- [x] Changelog amended with a mention like: "Added ``<country>`` by ``@pseudo`` (#)".

### CODE REVIEW

File: workalendar/africa/kenya.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5, 1, "Labour Day"),
+        (6, 1, "Madaraka Day"),
+        (10, 20, "Mashujaa Day"),
+        (12, 12, "Jamhuri Day"),
+        (12, 31, "New Years Eve"),
+    )
+
+    def get_fixed_holidays(self, year):
```
Comment (KidkArolis) [inline workalendar/africa/kenya.py:27]: Added this to dynamically introduce 'Moi Memorial Day' in 2020 and to rename some things based on year.

File: workalendar/africa/kenya.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+        # Boxing day renamed
+        boxing_day_label = "Boxing Day"
+        if year >= 2020:
+            boxing_day_label = "Utamaduni Day"
+        days.append((date(year, 12, 26), boxing_day_label))
+
+        return days
+
+    def get_shifted_holidays(self, dates):
```
Comment (KidkArolis) [inline workalendar/africa/kenya.py:47]: Added these next two methods to shift Sundays to Mondays, but I've excluded the Islamic days from being shifted.

**Comment by KidkArolis** [comment]
Fixed the linting.

File: Changelog.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## master (unreleased)
 
-Nothing here yet.
+- Added Kenyan calendar, by @KidkArolis (#???)
```
Comment (brunobord) [inline Changelog.md:5]:
```suggestion
- Added Kenyan calendar, by @KidkArolis (#484)
```

Pointing at this very pull-request is enough :)

File: workalendar/africa/kenya.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
+from copy import copy
+from datetime import timedelta, date
+
+from ..core import WesternCalendar, IslamicMixin, ChristianMixin
+from ..registry_tools import iso_register
+
+SUN = 6
```
Comment (brunobord) [inline workalendar/africa/kenya.py:7]: Sunday is already a core constant in workalendar.

```python
from ..core import SUN
```

File: workalendar/africa/kenya.py (hunk 1 above)
Comment (brunobord) [reply workalendar/africa/kenya.py:28]: that's the right way to go!

File: workalendar/africa/kenya.py (hunk 2 above)
Comment (brunobord) [reply workalendar/africa/kenya.py:48]: excellent!

**Comment by KidkArolis** [comment]
Amended

**Comment by brunobord** [comment]
The latest workalendar release (8.4.0) now integrates your Kenyan calendar. Happy upgrading! https://pypi.org/project/workalendar/8.4.0/

**Pull request merged**